src/interactive/repl.ts

```diff
@@ -276,6 +311,46 @@ async function startREPL(preserveFocus: boolean, showTerminal: boolean = true) {
     await juliaIsConnectedPromise.wait()
 }
 
+async function startREPLWithVersion() {
+    const versions = await getInstalledJuliaVersions()
+    const select = await vscode.window.showQuickPick(versions, {
+        placeHolder: 'Select version',
+        title: 'Start REPL with specific version',
+    })
+
+    if (select) {
+        await startREPL(false, true, select.Name)
+    }
+}
+
+async function getInstalledJuliaVersions() {
+    const juliaup = await g_juliaExecutablesFeature.getActiveJuliaupExecutableAsync()
```

Comment:
What happens if juliaup is not installed? Do we show an error message in that case?